dev-libs/girara/girara-0.3.5.ebuild

```diff
@@ -0,0 +1,48 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit meson virtualx
+
+DESCRIPTION="UI library that focuses on simplicity and minimalism"
+HOMEPAGE="https://pwmt.org/projects/girara/"
+
+if [[ ${PV} == *999 ]]; then
+	inherit git-r3
+	EGIT_REPO_URI="https://git.pwmt.org/pwmt/${PN}.git"
+	EGIT_BRANCH="develop"
+else
+	SRC_URI="https://pwmt.org/projects/girara/download/${P}.tar.xz"
+	KEYWORDS="~amd64 ~arm ~x86"
+fi
+
+LICENSE="ZLIB"
+SLOT="0"
+IUSE="doc libnotify test"
+
+RESTRICT="!test? ( test )"
+
+RDEPEND="dev-libs/glib:2
+	dev-libs/json-c:=
+	>=x11-libs/gtk+-3.20:3
+	libnotify? ( x11-libs/libnotify )"
+
+DEPEND="${RDEPEND}
+	doc? ( app-doc/doxygen )
```

Comment:
I imagine doxygen can go in BDEPEND and we don't run tests in cross so we could put check in there too.